sdk/translation/Azure.AI.Translation.Document/samples/CodeSamples/SampleTranslationWithAzureBlob.cs

```diff
@@ -0,0 +1,105 @@
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+/**
+FILE: SampleTranslationWithAzureBlob.cs
+DESCRIPTION:
+    This sample demonstrates how to use Azure Blob Storage to set up the necessary resources to create a translation
+    job. Run the sample to create containers, upload documents, and generate SAS tokens for the source/target
+    containers. Once the job is completed, use the storage library to download your documents locally.
+USAGE:
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_DOCUMENT_TRANSLATION_ENDPOINT - the endpoint to your Document Translation resource.
+    2) AZURE_DOCUMENT_TRANSLATION_KEY - your Document Translation API key.
+    3) AZURE_STORAGE_SOURCE_ENDPOINT - the endpoint to your Storage account
+    4) AZURE_STORAGE_ACCOUNT_NAME - the name of your storage account
+    5) AZURE_STORAGE_SOURCE_KEY - the shared access key to your storage account
+    Optional environment variables - if not set, they will be created for you
+    6) AZURE_STORAGE_SOURCE_CONTAINER_NAME - the name of your source container
+    7) AZURE_STORAGE_TARGET_CONTAINER_NAME - the name of your target container
+    8) AZURE_DOCUMENT_NAME - the name and file extension of your document in this directory
+        e.g. "mydocument.txt"
+**/
+
+namespace DocumentTranslatorSamples
+{
+    class SampleTranslationWithAzureBlob
+    {
+        public async static Task RunSampleAsync()
+        {
+            var endpoint = new Uri(Environment.GetEnvironmentVariable("AZURE_DOCUMENT_TRANSLATION_ENDPOINT"));
+            var key = Environment.GetEnvironmentVariable("AZURE_DOCUMENT_TRANSLATION_KEY");
+            var storageEndpoint = new Uri(Environment.GetEnvironmentVariable("AZURE_STORAGE_SOURCE_ENDPOINT"));
+            var storageAccountName = Environment.GetEnvironmentVariable("AZURE_STORAGE_ACCOUNT_NAME");
+            var storageAccountKey = Environment.GetEnvironmentVariable("AZURE_STORAGE_SOURCE_KEY");
+            var sourceContainerName = Environment.GetEnvironmentVariable("AZURE_STORAGE_SOURCE_CONTAINER_NAME");
+            var targetContainerName = Environment.GetEnvironmentVariable("AZURE_STORAGE_TARGET_CONTAINER_NAME");
+
+            // Create source and target storage containers
+            BlobServiceClient blobServiceClient = new BlobServiceClient(storageEndpoint, new StorageSharedKeyCredential(storageAccountName, storageAccountKey));
+            BlobContainerClient sourceContainerClient = await blobServiceClient.CreateBlobContainerAsync(sourceContainerName ?? "translation-source-container").ConfigureAwait(false);
+            BlobContainerClient targetContainerClient = await blobServiceClient.CreateBlobContainerAsync(targetContainerName ?? "translation-target-container").ConfigureAwait(false);
+
+            // Upload blob (file) to the source container
+            BlobClient srcBlobClient = sourceContainerClient.GetBlobClient("example_source_document.txt");
```

Comment:
Shouldn't this be AZURE_DOCUMENT_NAME?

Reply:
addressed that by adding AZURE_DOCUMENT_PATH as we need the whole path to not make the user put that document in the bin folder

Reply:
and added the files path to the readme